src/main/java/ch/wisv/areafiftylan/security/token/AuthenticationToken.java

```diff
@@ -0,0 +1,20 @@
+package ch.wisv.areafiftylan.security.token;
+
+import ch.wisv.areafiftylan.model.User;
+
+import javax.persistence.Entity;
+
+/**
+ * Created by Sille Kamoen on 6-5-16.
+ */
+@Entity
+public class AuthenticationToken extends Token {
```

Comment:
Please use a final EXPIRATION field for default expiration length.

Can be seen in https://github.com/AreaFiftyLAN/lancie-api/blob/master/src/main/java/ch/wisv/areafiftylan/security/token/Token.java#L14